pkg/koordlet/util/system/resctrl.go

```diff
@@ -427,6 +441,16 @@ func ReadResctrlSchemataRaw(schemataFile string, l3Num int) (*ResctrlSchemataRaw
 	return schemataRaw, nil
 }
 
+// GetCacheIds get cache ids from schemata file.
+// e.g. schemata=`L3:0=fff;1=fff\nMB:0=100;1=100\n` -> [0, 1]
+func GetCacheIds() ([]int, error) {
```

Comment:
the result can be cached, so that we don't need to read the file every time.

Reply:
I am attempting to use `sync.OnceValues` for caching cache IDs. However, this function was introduced in the standard library starting from Go 1.21. Should I temporarily place it in the `util` package and replace the function once Koordinator is upgraded to version 1.21?